### PR TITLE
support go in apiview

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Add GO in APIView supported languages

### CODE REVIEW

File: src/go/cmd/api_view.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func CreateAPIView(pkgDir, outputDir string) error {
 	if outputDir == "." {
 		outputDir = ""
 	}
-	filename := fmt.Sprintf("%s%s.json", outputDir, pkg.p.Name)
+	filename := fmt.Sprintf("%s/%s.json", outputDir, pkg.p.Name)
```
Comment (catalinaperalta) [inline src/go/cmd/api_view.go:95]: Could you do this instead: 
```
        if outputDir == "." {
		outputDir = ""
	} else if !strings.HasSuffix(outputDir, "/") {
		outputDir = fmt.Sprintf("%s/", outputDir)
	}
	filename := fmt.Sprintf("%s%s.json", outputDir, pkg.p.Name)
```

You can copy&paste this, I've already tested it.

**Review by pakrym — APPROVED** [review]

**Review by catalinaperalta — APPROVED** [review]
LGTM!

**Comment by praveenkuttappan** [comment]
I have deployed this change to staging and tested it.

**Pull request merged**